utils/browser.py

```diff
@@ -20,16 +21,18 @@
 
 from fixtures.pytest_store import store, write_line
 from utils import conf, tries
-from utils.log import logger
 from utils.path import data_path
 
 
+log = logging.getLogger('cfme.browser')
```

Comment:
For now, why not just use the standard logger, so that we actually see the mssages in artifactor and move to this when we have the new adapter?